src/st/no1/e_red_door.c

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+#include "common.h"
+
+INCLUDE_ASM("st/no1/nonmatchings/e_red_door", EntityIsNearPlayer);
```

Comment:
This file seems like it needs another split but doesn't have to be done now

Reply:
Yep, the OVL_EXPORT on the red door means it's hard to auto-split on it; I thought about adding that functionality but decided that there wasn't much reason to, since this is the only function that would need it, and we can manually split on our own.